Foundation/NSXMLDTDNode.swift

```diff
@@ -62,7 +62,7 @@ public class NSXMLDTDNode : NSXMLNode {
         }
     } //primitive
     
-    public override init(kind: NSXMLNodeKind, options: Int) {
+    public override init(kind: NSXMLNodeKind, options: NSXMLNodeOptionRawType) {
```

Comment:
That makes sense.  In that case, the type seems necessary. The reason being is that ```Int``` on 32-bit platforms is (as expected) a signed 32-bit integer.  In [NSXMLNodeOptions:105](https://github.com/apple/swift-corelibs-foundation/blob/master/Foundation/NSXMLNodeOptions.swift#L105) the upper bits are set to '1' (including bit 31) which is an overflow in that case.  

Because ```NSXMLNodeOptionRawType``` is a typealias to ```Int``` on 64-bit machines and ```Int64``` on 32-bit machines Swift doesn't require casts (to my knowledge) on 64-bit.  On 32-bit machines, it guarantees that ```NSXMLNodePreserveAll``` won't overflow (among other issues).